### PR TITLE
DOC-6040: Databases upgrade release notes

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

File: content/en/platform/corda/1.6/cenm/release-notes.md [hunk 1]
```diff
@@ -26,9 +26,37 @@ title: Release notes
...
-* Hibernate Validator has been upgraded to 6.2.5.Final.
-* Liquibase has been upgraded from 3.6.3 to 4.20.0.
-* Apache Tomcat has been upgraded from 9.0.80 to 9.0.81.
-* Bouncy Castle has been upgraded from 1.68 to 1.75.
+* Hibernate has been upgraded from 5.4.32.Final to 5.6.14.Final
+* Hibernate Validator has been upgraded to 6.2.5.Final
+* Apache Tomcat has been upgraded from 9.0.80 to 9.0.81
+* Bouncy Castle has been upgraded from 1.68 to 1.75
+
+* H2 has been upgraded from 1.4.197 to 2.1.214
```
Comment (tomstark99) [inline content/en/platform/corda/1.6/cenm/release-notes.md:34]: This should be changed to 2.2.214 as well (I haven't actually updated this yet, that is the corda version but the aim is to update to this version for cenm 1.6 as well)

File: content/en/platform/corda/1.6/cenm/release-notes.md [hunk 2]
```diff
@@ -26,9 +26,37 @@ title: Release notes
...
+
+  H2 database has been upgraded to version 2.2.224 primarily to address vulnerabilities reported in earlier versions of H2.
+  H2 is not a supported production database and should only be utilized for development and test purposes. For detailed information
+  regarding the differences between H2 version 1.4.197 used in 4.10 and below, and the new H2 version 2.2.224 implemented in 4.11,
```
Comment (tomstark99) [inline content/en/platform/corda/1.6/cenm/release-notes.md:38]: .. used in (CENM) 1.5.9 and below ...

*can omit '(CENM)' just for reference*

Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/release-notes.md:38]: Updated

**Review by tomstark99 — APPROVED** [review]
LGTM

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/1.6/cenm/release-notes.md [hunk 3]
```diff
@@ -26,9 +26,37 @@ title: Release notes
 ### Upgraded dependencies
 
 This release includes the following dependency version changes between CENM 1.5.9 and 1.6:
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/release-notes.md:28]: Added full stops.

File: content/en/platform/corda/1.6/cenm/release-notes.md [hunk 4]
```diff
@@ -26,9 +26,37 @@ title: Release notes
...
+
+    H2 2.x is unable to read database files created by older H2 versions. The recommended approach for upgrading an older database
+    involves exporting the data and subsequently re-importing it into a new version 2.x database. Further details on this
+    process are outlined on the [H2 Migration to 2.0](https://www.h2database.com/html/migration-to-v2.html) page.
+
+* Liquibase has been upgraded from 3.6.3 to 4.20.0
+
+  * Logging
+
+    In this version of Liquibase, all informational logging is directed to stderr, while stdout is used for logging SQL queries.
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/release-notes.md]: Changed to INFO-level logging

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**